dev/com.ibm.websphere.appserver.features/visibility/public/batchManagement-2.0/io.openliberty.batchManagement-2.0.feature

```diff
@@ -27,6 +27,6 @@ Subsystem-Name: Batch Management 2.0
 -files=bin/batchManager.bat, \
```

Comment:
If you don't have batchManager 1.0 also in your environment, batchManager scripts won't work I believe.  As such maybe batchManager shouldn't be marked beta, yet?  Is it needed for the CTS?